AENS.md

```diff
@@ -347,6 +376,30 @@ the name enters the `revoked` state. After a fixed timeout of
 2016 blocks, the name will be available for claiming again.
 
 
+### Subname
+
+```
+ ------------ ----
+| name       | 63 |
+ ------------ ----
+| definition |    |
+ ------------ ----
+```
+
+The `subname` transaction MUST be signed by the owner of the `name` entry.
+
+`subname` transaction is used to define a tree of subnames below `name` entry
+along with the pointers for each subname.
+
+A `definition` field is a mapping of `subname` to `pointers`, where the `pointers`
+for subname SHOULD NOT contain multiple entries with the same key.
+
+The execution of `subname` transaction first removes all subnames belonging to provided
```

Comment:
May be made clearer (as only observable outcome is root hash of tree - included in block header). Though minor.
```
The execution of `subname` transaction is semantically equivalent to ....
```

Reply:
fixed in latest commits